gitops/generate_build.go

```diff
@@ -97,7 +118,8 @@ func DetermineBuildExecution(component appstudiov1alpha1.Component, params []tek
 			{
 				Name: "registry-auth",
 				Secret: &corev1.SecretVolumeSource{
-					SecretName: "redhat-appstudio-registry-pull-secret",
+					SecretName: component.Name,
```

Comment:
Is it supposed to be different for each component or the same for all?

Reply:
The name will be same as that of the component - but the secret content will be the same if registry repo is the default well-known one.

Reply:
I see. Let's keep it as is